web-app/models/plant.js

```diff
@@ -0,0 +1,37 @@
+const { Sequelize } = require("sequelize");
+const sequelize = require("../app.js");
+const Moisture = require('./moisture.js');
+
+Plant = sequelize.define("plant", {
+  id: {
+    type: Sequelize.INTEGER,
+    autoIncrement: true,
+    allowNull: false,
+    primaryKey: true,
+  },
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  location: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  MAC: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  }
+});
+//Define 1-Many relationship
+Plant.hasMany(Moisture, {
+  foreignKey: 'plantId',
+  as: "moisture",
+  onDelete: 'cascade',
+  hooks: true,
+});
+Moisture.belongsTo(Plant, {
```

Comment:
This should be in the other data model once you have started to use the correct conventions `associate:`. I just realized the docs are extremely lacking in how to do this though so please follow the instructions in this Stack Overflow answer:
https://stackoverflow.com/questions/71037887/how-to-reference-models-in-junction-model-with-sequelize-6-and-7-cli-generate

Reply:
Actually I think this problem is related to the models format now that I look closer. Take a look at the models that CLI will generate, and follow that format. Here is a tutorial / overview on the process: https://levelup.gitconnected.com/getting-started-with-sequelize-cli-c33c797f05c6